helm/attestation-verifier/charts/aas/values.yaml

```diff
@@ -71,8 +71,12 @@ securityContext:
     capabilities:
       drop:
         - all
+    privileged: false
+    seccompProfile:
+      type: RuntimeDefault

```

Comment:
does RuntimeDefault seccomprofile conflicts with the above drop all capabilities?  I assume drop-all capability takes precedent since it's more restrictive?